python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/class_maps.schema.yml

```diff
@@ -59,3 +59,15 @@ keys:
                   type: str
                   description: IPv6 Access-List Name.
                   convert_types: [ int ]
+            dscp:
```

Comment:
It is not clear from the descriptions above that you can only have dscp _or_ cos _or_ ip etc. Please add a description on qos key making describing the restriction. I guess one of these must be set, but they are mutually exclusive?

Reply:
Updated description